vic/drivers/image/src/vic_populate_model_state.c

```diff
@@ -24,13 +24,15 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *****************************************************************************/
 
+#include <vic_def.h>
+#include <vic_run.h>
 #include <vic_driver_image.h>
```

Comment:
OK, I'll make this change.  I'll also make it to vic_populate_model_state() in cesm driver, since it has the same issue.